azure-pipelines.yml

```diff
@@ -190,12 +190,32 @@ jobs:
         # Only required for Integration Test. Version number contains date (e.g. 3.1.2.20211028-dev)
         WKVERSION="3.$LATEST_TAG-$(patchBuildNumberForDev)"
       else
+        # Line 194 to 198 to be commeneted out except when we need
+        # to test something from personal feature branch
+        echo "Generating V0 Integration Test Package for $BUILD_SOURCEBRANCHNAME"
+        LATEST_TAG=$(curl https://api.github.com/repos/Azure/azure-functions-python-worker/tags -s | jq '.[0].name' | sed 's/\"//g' |  cut -d'.' -f-2)
+        NUSPEC="pack\Microsoft.Azure.Functions.V0.PythonWorker.nuspec"
+        # Only required for Integration Test. Version number contains date (e.g. 3.1.2.20211028-dev)
+        WKVERSION="0.$LATEST_TAG-$(patchBuildNumberForDev)"
         echo "No Matching Release Tag For $BUILD_SOURCEBRANCH"
       fi
 
       echo "##vso[task.setvariable variable=nuspec_path]$NUSPEC"
       echo "##vso[task.setvariable variable=worker_version]$WKVERSION"
```

Comment:
they are here temporarily to test the metadata generation task, will be commented out/removed once the testing is done